app/src/main/java/net/squanchy/schedule/view/EventsAdapter.kt

```diff
@@ -0,0 +1,59 @@
+package net.squanchy.schedule.view
+
+import android.content.Context
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import net.squanchy.R
+import net.squanchy.schedule.domain.view.Event
+
+internal class EventsAdapter(context: Context) : RecyclerView.Adapter<EventViewHolder>() {
+
+    enum class ItemViewType {
+        TYPE_TALK,
+        TYPE_OTHER
+    }
+
+    private val layoutInflater: LayoutInflater
+    private var events = emptyList<Event>()
+    private var listener: ScheduleViewPagerAdapter.OnEventClickedListener? = null
```

Comment:
This is actually never null, it's passed in `updateWith` as non-null

Reply:
479360f48d2416fbe3115362dc066713df8b8385